content/applications/general/integrations/google_translate.rst

```diff
@@ -0,0 +1,123 @@
+================
+Google Translate
+================
+
+*Google Translate* can be used to translate user generated text in the Odoo chatter.
+
+Google API console
+==================
+
+A majority of the setup for integrating *Google Translate* into Odoo is done with the *Google API
+console*. Once the following processes are complete, an *API key* is created to input in Odoo.
+
+.. seealso::
+   `Google Translate setup on Google <https://cloud.google.com/translate/docs/setup>`_
+
+Create a new project
+--------------------
+
+To get started, go to the `Google API Console <https://console.developers.google.com>`_. Then, log
+in with a *Google Workspace* account, if there is one. If not, log in with a personal Gmail account
+(this should match the email address that has billing attached to it).
+
+Next, click :guilabel:`Create Project` on the far-right of the :guilabel:`OAuth consent screen`.
+
+.. tip::
+   If the *Google Developer Console* has existing projects, click the drop-down menu next to the
```

Comment:
The Google console is referred to as *Google API Console* throughout the doc, suggesting to keep the name consistent :)
```suggestion
   If the *Google API Console* has existing projects, click the drop-down menu next to the
```